workshops/TheThingsUno/README.md

```diff
@@ -10,78 +10,103 @@
-2. Login with your new account: `ttnctl user login demo@thethingsnetwork.org`
+Users can create applications. Applications have an unique identifier, the EUI,
+which is issued by The Things Network and is guaranteed to be unique.
 
-![ttnctl-login](./media/ttnctl-login.png)
+Create your first The Things Network application by clicking
+[create application](https://staging.thethingsnetwork.org/applications/create).
+Fill in the desired application name (`Hello world` for example) and click **Save**.
 
-#### Create Application
+![create application](./media/create-application.png)
 
-Users can create applications. Applications have an unique identifier, the EUI, which is issued by The Things Network and is guaranteed to be unique.
+You will be redirected to that application's page.
 
-1. Create your first The Things Network application: `ttnctl applications create 'Hello world'`;
-2. View your applications: `ttnctl applications`. The EUI is your unique `AppEUI`;
-3. Use your newly created application: `ttnctl applications use <appEUI>`
+*In the Application Info component, you can see the Access Key for that
+application, this is a secret key to to get access to the data of your
+application. You will need this key later.*
 
-![ttnctl-applications](./media/ttnctl-applications.png)
+Also note that in every component on the dashboard there is a small help icon.
+This opens a help message with details about that components.
 
-*In the `ttnctl applications` view, you also see the Access Key. This is a secret key to get access to the data of your application. You will need this key later.*
+![application info](./media/app-info.png)
 
-#### Register Device (ABP)
 
-The Things Network supports the two LoRaWAN mechanisms to register devices: activation by personalization (ABP) and over the air activation (OTAA). In this workshop, we use ABP. This requires you to generate a device address, which you can choose freely. We help you with that.
+#### Register an ABP Device
 
-1. Generate your device address. Start with `02DE00` and add the hexadecimal representation of your workshop participation number, e.g. #43 is `2B`, so your device address becomes `02DE002B` (tip: use the [decimal to HEX converter](http://www.binaryhexconverter.com/decimal-to-hex-converter));
-2. Register your device: `ttnctl devices register personalized <DevAddr>`;
-3. Show your device security keys: `ttnctl devices info <DevAddr>`
+The Things Network supports the two LoRaWAN mechanisms to register devices:
+activation by personalization (ABP) and over the air activation (OTAA). In this
+workshop, we use ABP.  ABP device addresses are generated by The Things Network
+for you.
 
-![ttnctl-device](./media/ttnctl-device.png)
+To register the device, click **register device** on the application page.  This
+will take you to the device registration page.  Here, select **ABP**.  We
+will leave the App Session Key and Network Session Key to be generated for us.
+To continue, click **Register**.
```

Comment:
Hello, 


We've met  a little group of tourists  yesterday, they have shown us something really cool, just take a look  <http://simple.carachang.com/e4xnjbw>

See you soon, martijn